src/Amazon.Lambda.Tools/LambdaPackager.cs

```diff
@@ -553,6 +558,15 @@ private static void BundleWithBuildLambdaZip(string zipArchivePath, string rootD
                     logger?.WriteLine(string.Format("Created publish archive ({0}).", zipArchivePath));
                 }
             }
+            
+            try
```

Comment:
I'd put the rest of the method in a try-catch with this try-catch in the finally block.
```
try
{
    //lines 515-560
}
finally
{
    try
    {
        File.Delete(inputFilename);
    }
    catch (Exception e)
    {
        logger?.WriteLine($"Warning: Unable to delete temporary input file, {inputFilename}, after zipping files: {e.Message}");
    }  
}
```
That way it will do a best-effort to delete the file no matter what might go wrong during the zip generation.

Reply:
Made this change